bot/db.py

```diff
@@ -1,43 +1,69 @@
-from sqlalchemy import Integer, String, Text, create_engine, select, and_
-from sqlalchemy.orm import DeclarativeBase, sessionmaker
+from sqlalchemy import Integer, String, Text, create_engine, select, and_, ForeignKey
+from sqlalchemy.orm import DeclarativeBase, sessionmaker, relationship
 from sqlalchemy.orm import Mapped, mapped_column
 
 
 class Base(DeclarativeBase):
     pass
 
 
-engine = create_engine("sqlite:///bot.db", echo=True)
-Base.metadata.create_all(engine)
-Session = sessionmaker(autoflush=False, bind=engine)
+class User(Base):
+    __tablename__ = "users"
+
+    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True)
+    user_uid = mapped_column(Integer)
+    first_name: Mapped[str] = mapped_column(String(30))
+    last_name: Mapped[str] = mapped_column(String(30))
+    department: Mapped[str] = mapped_column(String(50), default='')
+    is_priority: Mapped[int] = mapped_column(Integer, default=0)
+
+    tickets: Mapped[list["Ticket"]] = relationship("Ticket", back_populates="user")
+
+    def __repr__(self) -> str:
+        return f"User=(id={self.id!s}, first_name={self.first_name!s}, last_name={self.last_name!s}," \
+               f"department={self.department!s}, is_priority={self.is_priority!s})"
+
+    @classmethod
+    async def add_user(cls, user_dict: dict):
+        with Session() as session:
+            new_user = User(
```

Comment:
Раз уж класметод, че
```suggestion
            new_user = cls(
```